hutte.yml

```diff
@@ -3,11 +3,13 @@ version: 1.0
 # Shell script to run when pushing the source code to the scratch orgs.
 # It's a great place to automate tasks like permission set assignments or data loading.
 push_script: |
-  sfdx --version
-  sfdx force:source:push -f --loglevel fatal 1>/dev/null
+  sf --version
+  export SF_LOG_LEVEL=fatal
+  sf project deploy start --ignore-conflicts 1>/dev/null
```

Comment:
maybe add a -w 45?
also, I wonder if we really need/want to keep the 1>/dev/null - wouldn't we rather encourage customers to leave that out, in order to show meaningful push errors in the logs?

Reply:
Actually 1>/dev/null  redirects standard output, errors is 2 instead of 1.

Reply:
I have removed `1>/dev/null` after Friday's conversation, as well as the `fatal` log level as discussed.